packages/api/src/typeDefs/query.graphql

```diff
@@ -44,6 +44,14 @@ type StoreSearchResult {
   facets: [StoreFacet!]!
 }
 
+type StorePerson {
+  id: String
+  email: String
+  givenName: String
+  familyName: String
+  isAuthenticated: Boolean
```

Comment:
Why need this `isAuthenticated`? If you can get user property, the user is already authenticated, right?

Reply:
Good question 🤔 I believe so, but also this helps cause it's a boolean and we can quickly use it in places where we only need to check if it's authenticated

Reply:
I have the same feeling as @igorbrasileiro. This `isAuthenticated`, alongside the `person` field of type query brings 5 different states:
person exists, person is null, isAuthenticated is true, isAuthenticated is false, and isAuthenticated is null. 

I have a hard time trying to figure out what this isAuthenticated `null` means. Is is because the user is not authenticated, the user is not auth-able, something went wrong with the server etc.

To make things simpler, I'd either make `person` be of type `StorePerson!` and isAuthenticated be of type `Boolean!` or remove this `isAuthenticated` field and use the nullability of `person` field as indicator if the user is auth or not.